resources/views/auth/admission/index.blade.php

```diff
@@ -73,26 +72,10 @@
     <hr>
     <h6>Összesen: <b class="right">{{$applications->count()}} jelentkező</b></h6>
     @can('finalize', \App\Models\Application::class)
-        @if($applicationDeadline?->addWeeks(1) < now())
-            <div class="card" style="margin-top:20px">
-                <div class="card-content">
-                    <div class="row" style="margin:0">
-                        <form id="finalize-application-process" method="POST"
-                              action="{{route('admission.finalize')}}">
-                            @csrf
-                            <div class="col">
-                                Hogyha a felvételi eljárás befejeződött, akkor a felvett jelentkezőket itt lehet
-                                jóváhagyni.
-                                Ezzel együtt minden más felvételiző elutasításra, anyagai törlésre, valamint az összes
-                                felvételihez kapcsolódó (felvételiztető) jog elvételre kerül.
-                                A "Véglegesítve" és "Behívva" státuszú jelentkezőket előbb el kell utasítani, vagy fel
-                                kell venni.
-                            </div>
-                            <x-input.button class="red right" text="Felvételi lezárása"/>
-                        </form>
-                    </div>
-                </div>
-            </div>
+        @if($applicationDeadline && $applicationDeadline < now()))
```

Comment:
Here, there is an extra `)` which appears on the generated page.

Reply:
I don't have my laptop for the rest of the weekend, if you want to merge could you fix it?